parser_option.go

```diff
@@ -66,15 +66,15 @@ func WithExpirationRequired() ParserOption {
 	}
 }
 
-// WithAudience configures the validator to require the specified audience in
-// the `aud` claim. Validation will fail if the audience is not listed in the
-// token or the `aud` claim is missing.
+// WithAudience configures the validator to require ONE of the specified
```

Comment:
A `matchAll` option would be useful here for users who want to ensure ALL audiences are present. 
https://github.com/golang-jwt/jwt/issues/342#issuecomment-2295309922

Reply:
I believe https://github.com/golang-jwt/jwt/pull/427 does just that, likely the implementation we'll go with.

Reply:
I just saw that, thanks!